### DISCUSSION TITLE
How to fix the SpamAssassin sending score "-0.626	HK_RANDOM_ENVFROM"

### DISCUSSION THREAD

**Discussion by rolix202** [postalserver/postal#1702] [Help with using Postal]
How to fix the SpamAssassin sending score "-0.626	HK_RANDOM_ENVFROM	

Envelope sender username looks random

thats the error message

**Comment by willpower232** [comment]
Unfortunately you'll need to edit the tag in the database or try creating a different server in Postal. Have a look at https://github.com/postalserver/postal/discussions/1432#discussioncomment-1249742
